bin/ch/ch.cpp

```diff
@@ -196,48 +207,536 @@ static void CALLBACK PromiseContinuationCallback(JsValueRef task, void *callback
     MessageQueue * messageQueue = (MessageQueue *)callbackState;
 
     WScriptJsrt::CallbackMessage *msg = new WScriptJsrt::CallbackMessage(0, task);
+
+#if ENABLE_TTD
+    ChakraRTInterface::JsTTDNotifyHostCallbackCreatedOrCanceled(true, false, false, task, msg->GetId());
```

Comment:
Nit: comments for true/false arguments would improve readability like
JsTTDNotifyHostCallbackCreatedOrCanceled(/*isCreate*/ true, /*isCancel*/ false, /*isRepeating*/ false, task, msg->GetId())